### ISSUE TITLE
Database Submodule under Version-Control

### ISSUE THREAD

**Issue by ko4life-psko** [ko4life-net/Fire-Drake-Project-v1453#1]
Since there will be a lot of database editing, there is a need to automate the process of putting the db changes under git.
e.g. creating a script that will use MSSQL's exporter to dump all the database as alter scripts including content. More details regarding this can be found [here](https://ko4life.net/topic/1210-pinned-post-for-suggestion/?tab=comments#comment-6022).

The process should be relatively easy and will allow to track each table changes in a separate file, so new users don't need to restore a .bak file, rather run a script that will import the database from the generated script.

**Comment by brzezinski-grzegorz** [comment]
So maybe we should create additional folder with Database and post just scripts as *.sql files with information about them as comments?

**Comment by ko4life-psko** [comment]
> So maybe we should create additional folder with Database and post just scripts as *.sql files with information about them as comments?

Exactly, but the scripts should be generated automatically in another repository. Separating it into modules making it easier to collaborate.

**Comment by stevewgr** [comment]
In case you're still working on this, we implemented something similar in here: https://github.com/stevewgr/ko-db

basically we build the db from scripts with a script.
